### PR TITLE
fix/add_to_CNR_compatibility_for_RH7_RPM_pep

### PR DESCRIPTION
**Pull request by cesarjorgemartinez**

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
 - Add `from` based on fowarder header in log context
 - Check 401 status response before body content in retrieveSubserviceId
 - Set Nodejs 10 as minimum version in packages.json (effectively removing Nodev8 from supported versions)
+- Compatibility with RedHat 7 (or Centos 7) RPM. This is a absoluute requirement otherwise it fails
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:4]:
```suggestion
- Compatibility with RedHat 7 (or Centos 7) RPM
```

No need of over-hightlygint items in CNR. Let's try to be sharp and aseptic ;)

**Review by AlvaroVega — APPROVED** [review]
LGTM

**Comment by AlvaroVega** [comment]
But, what is the change introduced to achieve it ?

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
> But, what is the change introduced to achieve it ?

I think was the one in previous PR https://github.com/telefonicaid/fiware-pep-steelskin/pull/446 (merged time ago)

**Comment by cesarjorgemartinez** [comment]
The change is in PR 446, but, I missing to insert into the CNR... I sorry

**Pull request merged**